### PR TITLE
Add wallet.json

### PR DESCRIPTION
**Pull request by orblivion**
One question would be what directory it should go in.

### CODE REVIEW

File: v2/json/wallet.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            "type": "string"
+          }
+        }
+      }
+    },
+    "preferences": {
+      "description": "Timestamped application-level preferences. Values can be objects or of a primitive type.",
+      "$comment": "enable-sync is seen in example wallet. encrypt-on-disk is seen in example wallet. they both have a boolean `value` field. Do we want them explicitly defined here? local and shared seem to have at least a similar structure (type, value [yes, again], version), value being the free-form part. Should we define those here? Or can there be any key under preferences, and `value` be literally be anything in any form?",
+      "type": "object",
+      "additionalProperties": {
```
Comment (orblivion) [inline v2/json/wallet.json:104]: We have `shared` and `local` but I think as far as the sdk cares it's any key.

File: v2/json/wallet.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": ["address_generator", "certificates", "encrypted", "ledger", "modified_on", "name", "private_key", "public_key", "seed"],
+        "additionalProperties": false,
+        "properties": {
+          "address_generator": {
+            "description": "Higher level manager of either singular or deterministically generated addresses",
+            "type": "object",
+            "oneOf": [
```
Comment (orblivion) [inline v2/json/wallet.json:28]: We can actually enforce that `name=determinstic-chain` comes with `change` and `receiving`, and `single-address` does not.

**Comment by lyoshenka** [comment]
this looks good. you're right that its not clear where this code goes. the whole repo is focused on protobufs. the readme describes how to compile them, and so on. 

we're in the process of restructuring our repos a bit, so there's a chance your PR (or even this whole repo) will move elsewhere. i'll merge this for now and move it to a toplevel `jsonschema` dir

**Comment by orblivion** [comment]
Cool, just keep in mind that my PR in lbry-sdk depends on the location of the file:

https://github.com/lbryio/lbry-sdk/pull/3572/files#diff-c8a3e8308e908f1b3ec4c3fdf457d082ccc2c21f4c4863b8888c60b296e0a1c8R5

**Comment by orblivion** [comment]
Oh never mind, you already moved it. I'll update the PR now then.

**Pull request merged**